DateRangeBox: Add Getting Started with DateRangeBox guide (#5041)
DevExpress/devextreme-documentation: concepts/05 UI Components/DateRangeBox/00 Getting Started with DateRangeBox/00 Getting Started with DateRangeBox.md

#include tutorials-intro-installationnote

> The DateRangeBox is a UI component that allows users to specify or pick dates in the needed range.

Reviewer: "Specify" is not really different from "pick". If you "pick", you still "specify". So I wanted to change that to "enter".

"Dates in the needed range"... It's not that dates are in a range - the dates themselves define a range. So I changed that part a bit too. 

```suggestion
DateRangeBox is a UI component that allows users to enter or pick two dates (a date range).
```

This tutorial shows how to configure basic DateRangeBox features.

<div class="simulator-desktop-container" data-view="/Content/Applications/GettingStartedWith/DateRangeBox/index.html, /Content/Applications/GettingStartedWith/DateRangeBox/index.js, /Content/Applications/GettingStartedWith/DateRangeBox/index.css"></div>

> Each section in this tutorial covers a single configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-devextreme-daterangebox" target="_blank">getting-started-with-devextreme-daterangebox</a>.

Reviewer 2: ```suggestion
Each section in this tutorial describes a single configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-devextreme-daterangebox" target="_blank">getting-started-with-devextreme-daterangebox</a>.
```
